commit c133569132c8537107aee793d0a97007ca6a1464
Author: hap <happppp@users.noreply.github.com>
Date:   Wed May 11 14:34:48 2016 +0200

    v60: revert clang compile warning fix that broke conditional compile, added var to save_item to prevent that warning

diff --git a/src/devices/cpu/v60/v60.cpp b/src/devices/cpu/v60/v60.cpp
index 0a20284..ec33e3e 100644
--- a/src/devices/cpu/v60/v60.cpp
+++ b/src/devices/cpu/v60/v60.cpp
@@ -85,6 +85,7 @@ v60_device::v60_device(const machine_config &mconfig, const char *tag, device_t
 	: cpu_device(mconfig, V60, "V60", tag, owner, clock, "v60", __FILE__)
 	, m_program_config("program", ENDIANNESS_LITTLE, 16, 24, 0)
 	, m_io_config("io", ENDIANNESS_LITTLE, 16, 24, 0)
+	, m_fetch_xor(BYTE_XOR_LE(0))
 	, m_start_pc(0xfffff0)
 {
 	// Set m_PIR (Processor ID) for NEC m_ LSB is reserved to NEC,
@@ -97,6 +98,7 @@ v60_device::v60_device(const machine_config &mconfig, device_type type, const ch
 	: cpu_device(mconfig, type, name, tag, owner, clock, shortname, source)
 	, m_program_config("program", ENDIANNESS_LITTLE, 32, 32, 0)
 	, m_io_config("io", ENDIANNESS_LITTLE, 16, 24, 0)
+	, m_fetch_xor(BYTE4_XOR_LE(0))
 	, m_start_pc(0xfffffff0)
 {
 	// Set m_PIR (Processor ID) for NEC v70. LSB is reserved to NEC,
@@ -426,6 +428,7 @@ void v60_device::device_start()
 	m_direct = &m_program->direct();
 	m_io = &space(AS_IO);
 
+	save_item(NAME(m_fetch_xor));
 	save_item(NAME(m_reg));
 	save_item(NAME(m_irq_line));
 	save_item(NAME(m_nmi_line));
diff --git a/src/devices/cpu/v60/v60.h b/src/devices/cpu/v60/v60.h
index a7b2c88..acb1c70 100644
--- a/src/devices/cpu/v60/v60.h
+++ b/src/devices/cpu/v60/v60.h
@@ -151,6 +151,7 @@ private:
 	address_space_config m_program_config;
 	address_space_config m_io_config;
 
+	offs_t              m_fetch_xor;
 	offs_t              m_start_pc;
 	UINT32              m_reg[68];
 	struct {

